### DISCUSSION TITLE
Chrome/Firefox plugin fails to login but direct browser login does work. Help?

### DISCUSSION THREAD

**Discussion by emorneau** [dani-garcia/vaultwarden#3072] [Q&A] [answered]
See the below error which includes a 404 when trying to log at 09:14:01 after a restart. What is this "GET /<p..> [10] (web_files) => 404 Not Found"?

![Screen Shot 2022-12-30 at 9 16 41 AM](https://user-images.githubusercontent.com/12186126/210079884-9034b2e8-4bd2-4ee3-bc01-97d44655a986.png)

**Comment by emorneau** [comment]
Note: I confirmed my login creds works with the browser and it works against the same server endpoint.

**Comment by BlackDex** [comment, marked as answer]
Update to the latest version of Vaultwarden v1.27.0.
That will fix your _issue_

**Reply by emorneau** [reply]
You can mark this issue as resolved.. you rock! Thank you!

**Reply by tessus** [reply]
You can mark BlackDex's answer as answer. ;-)

P.S.: GH discussions still lack a bit of features. e.g. I would love to be able to resolve or close a discussion. Unfortunately one can only mark a reply as answered in the `Q&A` category. I hope gh will improve discussions in the future.
